test/controllers/urls_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class UrlControllerTest < ActionController::TestCase
+  def setup
+    @controller = UrlsController.new
+  end
+
+  test "redirects to the original url" do
+    url = Url.create(original_url: "www.google.com")
+    get :show, shortened_url: url.shortened_url
+    assert_equal 302, response.status
+    assert_equal 'http://test.hostwww.google.com', response.header["Location"]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.